skimage/util/_bounding_box_crop.py

```diff
@@ -0,0 +1,84 @@
+import numpy as np
+
+__all__ = ['bounding_box_crop']
+
+
+def bounding_box_crop(image, bounding_box, axes=None, copy=False):
```

Comment:
What about this function signature?:
```suggestion
def bounding_box_crop(image, tl_corner, br_corner=None, size=None, axes=None, copy=False):
```
where `tl_corner` is the top-letf corner (we may find a better argument name :sweat_smile:), `br_corner` is the bottom-right corner, `size` is the bounding box dimensions.
`br_corner` and `size` are optional but both can't be `None` in the same time and must be consistent if both provided.
Broadcasting can be made when size is an integer value, in this case the crop is a square (cube, or whatever...).

Reply:
Thanks for your suggestion, @rfezzani. I prefer `bounding_box, axes=None` because it's natively nD...